MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -49,6 +48,7 @@
 cur_dir = os.getcwd()
 static_dir = Path(os.path.join(cur_dir, "static/"))
 tmp_dir = Path(os.path.join(cur_dir, "split_tmp_videos/"))
+tmp_dir2 = Path(os.path.join(cur_dir, "split_tmp_audios/"))
```

Comment:
I would give it a more descriptive name like `audio_tmp_dir`, or could both audio and video go in the same directory, since they'd have different file extensions?